src/components/existing-request/existing-request-display.vue

```diff
@@ -20,7 +20,8 @@
           </v-col>
         </v-row>
       </div>
-      <names-gray-box
+
+      <NamesGrayBox
```

Comment:
This format (PascalCase instead of kebab-case) allows you to click F12 on this component and it will go to its implementation.